include/ginkgo/core/log/papi.hpp

```diff
@@ -12,11 +12,11 @@
 #if GKO_HAVE_PAPI_SDE
 
 
+#include <sde_lib.h>
 #include <cstddef>
 #include <iostream>
 #include <map>
 #include <mutex>
-#include <sde_lib.h>
```

Comment:
should it be changed?

Reply:
I think this is missing `format_header`. But I think we should just leave it, since it will most likely change again in #1466.